overlay/base/usr/lib/system/_os/system.py

```
@@ -375,3 +375,47 @@ def delete(glob: str):
+                if proc.wait():
+                    errors.append(
+                        subprocess.CalledProcessError(proc.returncode, proc.args)
+                    )
+
+                queue.remove(proc)
+
+    if cleanup is not None:
+        cleanup()
+
+    if errors:
+        raise ExceptionGroup("CalledProcessError", errors)  # noqa: F821
```

Ruff (0.14.5)

Unused `noqa` directive (unused: `F821`)

Remove unused `noqa` directive

(RUF100)